gravityflow/gravityflow
# Fix tests failing with Gravity Forms 2.5

## Description
Closes [Issue#101](https://github.com/gravityflow/backlog/issues/101)

Fix the failing Unit Tests and Acceptance Tests, to conform to the new release of Gravity Forms (2.5).

## Testing instructions
1. Tests on `master` are failing with Gravity Forms 2.5
2. Switch to this branch and confirm all unit tests and acceptance tests pass fine.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
